Packs/Whois/ReleaseNotes/1_4_15.md

```diff
@@ -0,0 +1,6 @@
+#### Integrations
+##### Whois
+- Added support for Rate Limit handling with **rate_limit_retry_count** instance configuration field, and a command param. Default value: 3 times.
```

Comment:
Combine the 3 lines as follows:
- Added support for Rate Limit handling with the following instance configuration fields and command parameters:
   - *rate_limit_retry_count* - Default: 3 times.
   - *rate_limit_wait_seconds* - Default: 120 seconds.
   - *rate_limit_errors_suppressed* - Default: False.